packages/preload/src/index.ts

```diff
@@ -880,13 +880,15 @@ function initExposure(): void {
       log: LogFunction,
       warn: LogFunction,
       error: LogFunction,
+      end: () => void,
```

Comment:
I think we have a lot of code not related to the return of an id in an execution flow but around logger, the end of a task